common.py

```diff
@@ -76,6 +77,34 @@ class IperfOutput():
     command: str
     result: dict
 
+@dataclass
+class PluginOutput():
+    plugin_metadata: dict
+    command: str
+    result: dict
+    name: str
+
+@dataclass
+class TftAggregateOutput():
+    iperf_output: IperfOutput = None
+    measureCPU_client_output: Optional[PluginOutput] = None
+    measureCPU_server_output: Optional[PluginOutput] = None
+    measurePower_client_output: Optional[PluginOutput] = None
+    measurePower_server_output: Optional[PluginOutput] = None
+
+    def dump_to_dict(self) -> dict:
```

Comment:
Yeah I was thinking this would be a better way to do this as well. I had a previous iteration that required something a bit more involved, but I think a list of plugins would make more sense. I will add this